Automattic/jetpack
# Dashboard: Prevent console warnings when toggling on some Jetpack settings

Fixes https://github.com/Automattic/jetpack/issues/40086

## Proposed changes:

* `isModuleActivated` requires a module name, and the way it was being used in order to determine a value for the checked property within `ToggleControl`, nothing was being passed in. As such, it was returning undefined on any toggle change (I tested with all toggles to confirm). The best option would be to remove it, which is done here.
* This helps prevent a console warning when toggling on some Jetpack settings.



### Other information:


## Jetpack product discussion

https://github.com/Automattic/jetpack/issues/40086

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

You may need script debug enabled to see the console warnings: `define( 'SCRIPT_DEBUG', true );` in `wp-config.php`.

To replicate the issue:

* On a test site, open up the Jetpack Settings page. Not all toggles result in the warning, but the Sharing buttons toggle on the Sharing page does: `/wp-admin/admin.php?page=jetpack#/sharing`. 
* Open up the browser console, and enable Sharing buttons via the toggle.
* You should see a console warning related to `A component is changing an uncontrolled input to be controlled.`.

To test the fix:

* Apply this PR, and follow the same steps as above. There should be no warnings in the browser console.
